src/utils/trust.details.ts

```diff
@@ -147,23 +162,37 @@ export const postTrustDetails = async (req: Request, res: Response, next: NextFu
     }
 
     //  if present, get existing trust from session (as it might have attached trustees)
-    const trust = getTrustByIdFromApp(appData, details.trust_id);
+    let trust;
+    if (isReview) {
+      trust = getReviewTrustById(appData, details.trust_id);
+    } else {
+      trust = getTrustByIdFromApp(appData, details.trust_id);
```

Comment:
getTrustByIdFromAppData would be a better name for getTrustByIdFromApp

Reply:
that is an already existing method name that registration/update use so would rather not refator.